setup.py

```diff
@@ -67,8 +67,8 @@ def get_version():
             "qblox-instruments==0.7.0",
             "qcodes",
             "pyvisa-py==0.5.3",
-            "qm-qua>=1.0.1",
-            "qualang-tools>=0.13.1",
+            "qm-qua==1.1.1",
```

Comment:
That's fine with me, however what happens if they release a `qm-qua 1.1.2` that breaks something in the driver? The idea of pinning to the exact version with `==` was to avoid such issues as it ensures that we are using the version that works.

Of course it is not optimal but the main issues are:
1. It appears that these libraries are a bit unstable and it is very likely that new releases contain breaking changes,
2. Driver coverage in CI is very low and unless we use the label tests which run on the QPUs (which are also not available sometimes), driver code will not be tested. This makes it hard to spot these breaking changes.